src/rpc/handlers/LedgerData.hpp

```diff
@@ -73,9 +76,13 @@ class LedgerDataHandler {
         bool validated = true;
     };
 
-    // TODO: Clio does not implement "type" filter
-    // outOfOrder only for clio, there is no document, traverse via seq diff
-    // outOfOrder implementation is copied from old rpc handler
+    /**
+     * @brief A struct to hold the input data for the command
+     *
+     * @note TODO: Clio does not implement `type` filter
```

Comment:
Maybe add link to the related issue?

Reply:
Actually, we support `type` filter. This looks like an old TODO that slipped through. Removing it 👍